### ISSUE TITLE
Forms throws error if origami helpers included

### ISSUE THREAD

**Issue by BorntraegerMarc** [hotforfeature/origami#23]
When I add a form field with: `emitChanges ironControl` For example:
```
            <paper-input
              [(ngModel)]="loginForm.username"
              label="asdf"
              emitChanges ironControl
              type="text"
              id="username"
              class="form-control"
              name="username" #username></paper-input>
```

But I import the standard polymer elements with:
`import { IronElementsModule, PaperElementsModule, AppElementsModule } from '@codebakery/origami/lib/collections';
`

Then I get the following error:
```
zone.js:642 Unhandled Promise rejection: More than one custom value accessor matches form control with unspecified name attribute ; Zone: <root> ; Task: Promise.then ; Value: Error: More than one custom value accessor matches form control with unspecified name attribute
```

But the docs say "[emitChanges] and [ironControl] are not needed for paper elements" -> So question:
1. Do we ether say that these controls MUST NOT be added
2. Or is this a bug?

**Comment by hotforfeature** [comment]
The error is correct, because including the collection module adds a new directive, [PaperInputControl](https://github.com/hotforfeature/origami/blob/master/src/collections/paper-elements/paper-input-controls.ts) that selects `<paper-input>` so you don't have to add `[ironControl]`.

So in this example, you have both `PaperInputControl` and `IronControlDirective` trying to handle this one input, which is where the error comes from.

I should update the docs to more directly specify that you should not add `[emitChanges]` or `[ironControl]` when using a collections module.

**Comment by hotforfeature** [comment]
Fixed with 9ef37e5
